src/metabase/email/new_user_invite.mustache

```diff
@@ -1,8 +1,10 @@
 {{> metabase/email/_header }}
-  <div style="padding: 2em 4em; border: 1px solid #dddddd; border-radius: 2px; background-color: white; box-shadow: 0 1px 2px rgba(0, 0, 0, .08); text-align: center;">
+  <div style="text-align: center;">
     <div style="padding-bottom: 1em;">
-      <h2 style="font-weight: normal; color: #4C545B;line-height: 1.65rem;">{{invitedName}}, you're invited to {{company}}'s&nbsp;Metabase</h2>
-      <h4 style="font-weight: normal;"><a style="color: #4A90E2; text-decoration: none;" href="mailto:{{invitorEmail}}">{{invitorName}} ({{invitorEmail}})</a> invited you to join them.</h4>
+      <h2 style="font-weight: normal; color: #4C545B;line-height: 1.65rem;">{{invitorName}} wants you to join them on {{company}}'s&nbsp;Metabase</h2>
+
+      <!--h4 style="font-weight: normal;"><a style="color: #4A90E2; text-decoration: none;" href="mailto:{{invitorEmail}}">{{invitorName}} ({{invitorEmail}})</a> invited you to join them.</h4-->
```

Comment:
There's something goofy going on here with the HTML comments and and the `h4` tag.

Reply:
Whoops. Removed.